Bottom: 434a8a3024703487efd44158755c3b2d8567063f
Top:    1ec973cc2f9d39d94260535b52a1bb87f27882a4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-04 23:42:40 +0000

Refresh of play-with-render-to-texture-st

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 119fc11..556ab99 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1965,6 +1965,7 @@ cleanup_fbo_layer (GHidPort *port)
 void
 ghid_draw_everything (BoxTypePtr drawn_area)
 {
+  render_priv *priv = gport->render_priv;
   int i, ngroups;
   /* This is the list of layer groups we will draw.  */
   int do_group[MAX_LAYER];
